### ISSUE TITLE
Filtering Log Improvements - MENU

### ISSUE THREAD

**Issue by Nzzyme** [AdguardTeam/AdguardForAndroid#1950]
Hello,

I have a few suggestions with regards to the overall UI and usability of the Filtering Log. These are generic requests which are not related to any particular Android version, filters, etc. I have divided the requests/suggestions into three parts: Menu, Log Entry View, Long Tap View. I will be posting three different issues & will start with the first one - MENU

**MENU**:

I was thinking it is good that we have few additional options on the Filtering Log page as Menu options. Is it possible to include a menu on the Filtering Log page that include these options:

1. Show Blocked/Unblocked Requests Only
2. Sort on data size (sent/received) / Time (Oldest/recent)
3. Clear Log
4. Pause Log
5. Group by Name - Tapping this option would group multiple log entries from the same app in one group in the main list. Good if there are many entries repeatedly by the same app.
6. View - Can be Basic or Detailed if my second Issue (Log Entry View) is implemented in another thread.
7. Export Log(s) - Not sure if this might be needed but if you feel the need to.

I will soon be posting another issue for "Log Entry View" with different set of suggestions.

Thank you!

**Comment by Versty** [comment]
@Nzzyme Hi!
This concept is partially implemented, will be considered further in https://github.com/AdguardTeam/AdguardForAndroid/issues/3234

**Comment by TPS** [comment]
Trying again in https://github.com/AdguardTeam/AdguardForAndroid/issues/5849
